client/scripts/app.ts

```diff
@@ -305,7 +328,7 @@ export async function initChain(): Promise<void> {
   m.redraw();
 }
 
-export function initCommunity(communityId: string): Promise<void> {
+export function initCommunity(communityId: string): Promise<boolean> {
```

Comment:
what is being returned?

Reply:
This merely returns a Promise<boolean> because the function it calls, selectCommunity (which previously returned Promise<void>) has been altered to return a Promise<boolean>. Want me to add docs or..?